<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoindreQuitterWindow</class>
 <widget class="QMainWindow" name="JoindreQuitterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblAdresseServeur">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse du serveur :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAdresseServeur">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPort">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblPort">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Port :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnJoindre">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>170</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Joindre</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lbParties">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>31</y>
      <width>411</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblParties">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>401</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Liste des parties:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNouvellePartie">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Créer une nouvelle partie</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAfficherParties">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher les parties hébergées par le serveur</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
